cylc/flow/suite_files.py

```diff
@@ -992,6 +992,7 @@ def get_rsync_rund_cmd(src, dst, reinstall=False, dry_run=False):
         '.svn',
         '.cylcignore',
         'log',
+        'opt/rose-suite-cylc-install.conf',
```

Comment:
Just wondering if the intention is for anything else to be kept in the folder `opt`? As it stands, the empty `opt` folder will still be copied.

Reply:
Yes. Opt can contain other optional configs which we probably want to install, unless we choose to make cylc-rose do it, just as https://github.com/cylc/cylc-rose/issues/36 suggests installing `rose-suite.conf`.
IMO this is reasonable, since it is more consistent.

Reply:
On Element @oliver-sanders  said no:
> I think cylc install should exclude (via the default rsync exclude options):
>    ``rose-suite.conf``
>    ``opts/rose-suite-cylc-install.conf``
> The first file cylc-rose will need to copy into place.
> The second file cylc-rose creates/modifies as part of the post_install hook.

Reply:
Still need to exclude the `rose-suite.conf`?

Reply:
Ah wait, there's another parallel PR which adds the `rose-suite.conf` that will conflict with this one?